deploy/evmos/000_deploy_MiniChefV2.ts

```diff
@@ -0,0 +1,69 @@
+import { BIG_NUMBER_1E18 } from "../../test/testUtils"
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { MiniChefV2 } from "../../build/typechain"
+import { ethers } from "hardhat"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+    const { deployments, getNamedAccounts, getChainId } = hre
+    const { deploy, get, execute, getOrNull, log } = deployments
+    const { deployer } = await getNamedAccounts()
+
+    const miniChef = await getOrNull("MiniChefV2")
+    if (miniChef) {
+        log(`Reusing MiniChefV2 at ${miniChef.address}`)
+    } else {
+        // Deploy retroactive vesting contract for airdrops
+        await deploy("MiniChefV2", {
+            from: deployer,
+            log: true,
+            skipIfAlreadyDeployed: true,
+            args: [(await get("SDL")).address],
+        })
+
+        const minichef: MiniChefV2 = await ethers.getContract("MiniChefV2")
+
+        // Total LM rewards is 30,000,000 but only 12,500,000 is allocated in the beginning
+        // Evmos's portion is 500_000
+        const TOTAL_LM_REWARDS = BIG_NUMBER_1E18.mul(500_000)
+        // 2 months (8 weeks)
+        const lmRewardsPerSecond = TOTAL_LM_REWARDS.div(2 * 4 * 7 * 24 * 3600)
+
+        const batchCall = [
+            await minichef.populateTransaction.setSaddlePerSecond(lmRewardsPerSecond),
+            await minichef.populateTransaction.add(
+                100,
+                "0x1275203FB58Fc25bC6963B13C2a1ED1541563aF0", // evmos 3pool(USDC)
```

Comment:
Since this is a constant we already have saved, we should get LP token addresses using `get`

Reply:
resolved in https://github.com/saddle-finance/saddle-contract/pull/589/commits/bb6d726b3c8cf19f92cff9fb5625e41d5ec5970d